### ISSUE TITLE
Toastr not displaying again

### ISSUE THREAD

**Issue by PratibhaP** [jesusoterogomez/react-notify-toast#48]

**Comment by PratibhaP** [comment]
I have a created one http request and in catch used toastr.
It shows me proper toastr once there is error response from server. But if i have created one more same http request, I can see server error in network but toastr not showing again,
Here is the code

http.post(`***/api/auth`, {
				email: 'demo@gmail.com',
				password: 'password'
			})
			.then((response) => {
				// console.log(response);
				route('/something');
			}).catch(() => {
				let toastrColor = { background: '#FF0000', text: "#FFFFFF" };
			  notify.show('Error message', 'error', 10000, toastrColor);
			});

**Comment by jesusoterogomez** [comment]
Do you have additional information on the error that you get in the console? (when it doesn't display anymore)

Is it a problem with the DOM element not being found or you lose the reference to `notify` if the error happens twice?

**Comment by PratibhaP** [comment]
No, I am not getting any error, but it is not showing toastr again, may be the context of previous toastr it not destoryed, it just hidden, I guess.
